.gitignore

```diff
@@ -1,4 +1,5 @@
 **/.DS_Store
 .idea/
 dist/
-node_modules/
\ No newline at end of file
+node_modules/
+package-lock.json
```

Comment:
Don't ignore this, it can cause some weird bugs

Reply:
Do we have to commit `package-lock.json` then?

Reply:
We do, at least npm recommends that. It keeps tracks of the versions of subdependencies and the order of installation

Reply:
Noted. I'll commit it then